src/http/kv/create_namespace.rs

```diff
@@ -0,0 +1,32 @@
+use cloudflare::apiclient::APIClient;
```

Comment:
I guess I should file a bug upstream, but this should be `ApiClient`

Reply:
We're happy to rename the types